components/CapBar/index.js

```diff
@@ -28,15 +28,16 @@ const CapBar = ({
   current = current > cap ? cap : current;
 
   return (
-    <div className='w-100'>
-      <div className='d-flex flex-row justify-content-between'>
+    <div className="w-100">
+      <div className="d-flex flex-row justify-content-between">
         <SecondaryText color={colors.text} fontSize={labelConfig.fontSize}>
           {copies.current}
         </SecondaryText>
         <Title fontSize={statsConfig.fontSize} lineHeight={20}>
           {loading
-            ? 'Loading...'
-            : `${displayCurrent ? displayCurrent : `${formatAmount(current)} VEX-VET`
+            ? "Loading..."
+            : `${
+                displayCurrent ? displayCurrent : `$${formatAmount(current)}`
               }`}
```

Comment:
While I prefer `"` to `'`, this multiline split is dubious. I assume this was your local prettier running again?